python/main.py

```diff
@@ -114,10 +114,10 @@ def getKeyword(item_id: int):
 
 @app.get("/image/{image_filename}")
 async def get_image(image_filename):
-
+    
     # Create image path
     image = images / image_filename
-
+    print(image)
```

Comment:
debug?

Reply:
Ah right, I used it for debugging. I didn't notice it is still there. Thank you for pointing it out!